astrobase/lcmath.py

```diff
@@ -1341,6 +1342,11 @@ def phase_bin_magseries_with_errs(phases, mags, errs,
         The minimum number of elements required per bin to include it in the
         output.
 
+    weights : np.array or None
+        Optional weight vector to be applied during binning. If if is passed,
+        `np.average` is used to bin, rather than `np.median`. A good choice
+        would be to pass weights=1/errs**2, to weight by the inverse variance.
```

Comment:
this should be:

```suggestion
        would be to pass ``weights=1/errs**2``, to weight by the inverse variance.
```

to stop Sphinx from thinking it's bold text markup.